projects/ccf-eui/src/app/app.component.theme.scss

```diff
@@ -20,4 +20,10 @@
     color: mat.get-color-from-palette($foreground, contrast-icon);
     background-color: mat.get-color-from-palette($background, modal);
   }
+
+  .organ-selector {
+    .cls-2-pelvis {
```

Comment:
This is bizarre. Please update the pelvis svg so you don't need this oddity. No other icons need this.